src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/HealthCertifiedPersonViewController.swift

```diff
@@ -153,16 +148,16 @@ class HealthCertifiedPersonViewController: UIViewController, UITableViewDataSour
 	// MARK: - Protocol UITableViewDelegate
 
 	func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
-		// we are only interested in QRCode cell once if the traitCollectionDidChange - to update gradientHeightConstraint
+		// we are only interested in admission state cell once if the traitCollectionDidChange - to update gradientHeightConstraint
 		guard
 			didCalculateGradientHeight == false,
-			HealthCertifiedPersonViewModel.TableViewSection.map(indexPath.section) == .qrCode
+			HealthCertifiedPersonViewModel.TableViewSection.map(indexPath.section) == .admissionState
```

Comment:
Hm, I think this would break if the admission state is hidden. Maybe we need the info from the view model which one is the topmost cell?

Reply:
Good catch!